ci/do_ci.sh

```diff
@@ -377,7 +377,7 @@ case $CI_TARGET in
         # fi
         ;;
 
-    check_and_fix_proto_format)
+    format-api|check_and_fix_proto_format)
```

Comment:
how about `format_api`?

Reply:
apologies - just saw this as i pushed through

ive been moving these to `kebab-case`